### PR TITLE
fix: [ANDROAPP-5930] remove old unnecessary header and modify edit en…

### PR DESCRIPTION
**Pull request by xavimolloy**
…rollment corners and header

## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://dhis2.atlassian.net/browse/ANDROAPP-

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] 11.X - 13.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/androidTest/java/org/dhis2/usescases/searchte/SearchTETest.kt [hunk 1]
```diff
@@ -107,6 +109,7 @@ class SearchTETest : BaseTest() {
             openNextSearchParameter("Last name")
             typeOnNextSearchTextParameter(lastName)
             clickOnSearch()
+            composeTestRule.waitForIdle()
```
Comment (ferdyrod) [inline app/src/androidTest/java/org/dhis2/usescases/searchte/SearchTETest.kt:112]: Could this potentially introduces flakines to the test??

**Review by ferdyrod — APPROVED** [review]

**Review by andresmr — CHANGES_REQUESTED** [review]
please check ci

File: commons/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -259,7 +259,7 @@
         <item quantity="other">enrollment</item>
     </plurals>
     <string name="dialog_layer_enrollment_coordinates_v2">Show %s coordinates</string>
-
+    <string name="empty_enrollment">%s</string>
```
Comment (Balcan) [inline commons/src/main/res/values/strings.xml:262]: This is has no sense. Remove this empty resource.

File: commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt [hunk 3]
```diff
@@ -58,6 +58,31 @@ class ResourceManager(
         }
     }
 
+    fun defaultEnrollmentLabel(
```
Comment (Balcan) [inline commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt:61]: Keep this method and return the enrollmentLabel. Capitalize when needed. Also, `formatWithEnrollmentLabel` could call it to return get the enrollment label and we reduce duplication.

File: commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt [hunk 4]
```diff
@@ -58,6 +58,31 @@ class ResourceManager(
...
+            D2Manager.getD2().programModule().programs().uid(programUid).blockingGet()
+                ?.enrollmentLabel()
+        } catch (e: Exception) {
+            null
+        } ?: getPlural(R.plurals.enrollment, quantity)
+
+        return enrollmentLabel.capitalize(Locale.current)
+    }
+
+    fun defaultEnrollmentLabel(
```
Comment (Balcan) [inline commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt:75]: remove this method

File: app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt [hunk 5]
```diff
@@ -316,36 +311,13 @@ class EnrollmentActivity : ActivityGlobalAbstract(), EnrollmentView {
...
-                Glide.with(this).load(File(teiInfo.profileImage))
-                    .transition(DrawableTransitionOptions.withCrossFade())
-                    .transform(CircleCrop())
-                    .into(binding.teiDataHeader.teiImage)
-                binding.teiDataHeader.teiImage.setOnClickListener {
-                    presenter.onTeiImageHeaderClick()
-                }
-            }
+            binding.title.text =
+                resourceManager.formatWithEnrollmentLabel(
```
Comment (Balcan) [inline app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentActivity.kt:315]: It makes no sense to use an empty resource. Just call the defaultEnrollmentLabel method you added.

**Review by Balcan — CHANGES_REQUESTED** [review]

File: commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt (hunk 4 above)
Comment (xavimolloy) [reply commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt]: nice catch!

File: commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt (hunk 3 above)
Comment (xavimolloy) [reply commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt:48]: you are right that was my original idea but I must have gotten distracted halfway through

**Review by Balcan — APPROVED** [review]

**Review by andresmr — APPROVED** [review]

**Pull request merged**